### PR TITLE
Add python 3.5.2

### PR DESCRIPTION
**Pull request by kyle-bookbub**
Overwrites the existing 3.5.9 version with 3.5.2. Trying to add a new version proved tricky given the needed change the path file and the various places python35 is used. Replacing it in place seemed better. If we need another app with "latest" master of nixpkgs and another specific 3.5.x version we can make another fork I think.

### CODE REVIEW

**Comment by kyle-bookbub** [comment]
I tested this locally in laminar. Deleting the current venv environment files and then restarting it while running this nixpkgs loads up 3.5.2 as expected. It's not that huge of a build compared to other runtimes.

File: pkgs/development/interpreters/python/cpython/3.5/no-ldconfig.patch [hunk 1]
```diff
@@ -12,53 +12,35 @@ diff --git a/Lib/ctypes/util.py b/Lib/ctypes/util.py
...
--
--        c_compiler = shutil.which('gcc')
--        if not c_compiler:
--            c_compiler = shutil.which('cc')
--        if not c_compiler:
--            # No C compiler available, give up
--            return None
--
--        temp = tempfile.NamedTemporaryFile()
+-        expr = r'[^\(\)\s]*lib%s\.[^\(\)\s]*' % re.escape(name)
```
Comment (kyle-bookbub) [inline pkgs/development/interpreters/python/cpython/3.5/no-ldconfig.patch:19]: This changed between 3.5.2 and 3.5.9. I downloaded 3.5.2 locally and created this git patch manually. Seems to be the only difference.

**Review by nbr — APPROVED** [review]
This is great. It took me 90s to build the first time locally; I don't think this is ideal but may be tolerable given the alternatives are heavyweight and the user base for Laminar is probably small.

**Comment by kyle-bookbub** [comment]
Given we are doing a 20-40 minute build in Mockingjay for nodejs, I think a one-time tax of a few minutes is alright.

**Pull request merged**